Assets/MixedRealityToolkit.Services/InputSystem/InputSystemGlobalListener.cs

```diff
@@ -13,6 +14,10 @@ public class InputSystemGlobalListener : MonoBehaviour
     {
         private bool lateInitialize = true;
 
+        /// <summary>
+        /// Prefer using EnsureInputSystemValid(), which will only await if the input system isn't
+        /// yet valid.
+        /// </summary>
         protected readonly WaitUntil WaitUntilInputSystemValid = new WaitUntil(() => MixedRealityToolkit.InputSystem != null);
```

Comment:
Looks good. Should we mark this one as deprecated so users don't take dependencies on it?

Reply:
Done